packages/@aws-cdk/aws-autoscaling/README.md

```diff
@@ -9,14 +9,17 @@ An `AutoScalingGroup` represents a number of instances on which you run your cod
 pick the size of the fleet, the instance type and the OS image:
 
 ```ts
+import autoscaling = require('@aws-cdk/aws-autoscaling');
 import ec2 = require('@aws-cdk/aws-ec2');
 
-new ec2.AutoScalingGroup(stack, 'ASG', {
+const vpc = new ec2.VpcNetwork(stack, 'VPC', {
+    maxAZs: 3
```

Comment:
No need to limit AZs in this example, just leave `new ec2.VpcNetwork(stack, 'VPC')`

Reply:
Ok, I will remove it.